unbxdAutosuggest.css

```diff
@@ -219,18 +219,20 @@
     max-width: 100px;
     max-height: 100px;
     float: none;
+    margin: 0 auto;
```

Comment:
As discussed let's use new CSS file, otherwise this changes will effect prod sites probably.

Reply:
done